dynamo-store.js

```diff
@@ -594,8 +628,9 @@ function make_intern() {
 
             let sk = indexdefkey.sort
             if (null != sk && null != fq[sk]) {
-              listreq.KeyConditionExpression += ` and #${sk}n = :${sk}i`
-              listreq.ExpressionAttributeValues[`:${sk}i`] = fq[sk]
+              let fq_op = get_op(fq[sk])
+              listreq.KeyConditionExpression += null == fq_op ? ` and #${sk}n = :${sk}i` : ` and #${sk}n ${fq_op.op} :${sk}i`
```

Comment:
create an explicit var for fq_op.op - this makes the code more self documenting

Reply:
Agreed. Done.